gitops-wg/README.MD

```diff
@@ -1,3 +1,89 @@
+This, combined with the recommendation by the Cloud Native Computing Foundation (CNCF) [user community to adopt Flux](https://radar.cncf.io/2020-06-continuous-delivery), made it clear that GitOps is fast becoming the methodology of choice for operating modern cloud native infrastructure and applications.
+The CNCF user community reported that development, DevOps, and operations teams who adopt GitOps tooling and follow best practices experience improvements in productivity, stability, reliability, and security for their cloud native environments.
+
+## The What and Why of GitOps
+
+If you are new to GitOps, [it builds and iterates](https://www.weave.works/blog/gitops-operations-by-pull-request) on ideas drawn from DevOps and Infrastructure as Code that started with [Martin Fowler’s comprehensive Continuous Integration overview](https://martinfowler.com/articles/continuousIntegration.html) and provides the freedom to choose the tools that you need for your specific use cases.
+
+Individuals, teams, and organizations who implement GitOps experience many benefits, including:
+
+- Increased Developer & Operational Productivity
+- Enhanced Developer Experience
+- Improved Stability
+- Higher Reliability
+- Consistency and Standardization
+- Stronger Security Guarantees
+
+## GitOps Principles
+
+The group’s initial task – to "clearly define a vendor-neutral, principle-led meaning of GitOps" – is expressed as GitOps _Principles_, which are the foundation of GitOps practices.
+
+This aims to give participants in the cloud native ecosystem a common understanding of GitOps systems based on shared principles, rather than a matter of individual opinion or implementation preference.
+
+🚧 The GitOps Principles are currently a work in progress. For current status, follow [PR #48](https://github.com/gitops-working-group/gitops-working-group/pull/48).
+
+🕠 In the meantime, you may refer to the initial [Draft Definition](https://docs.google.com/document/d/11EZfvB2FFI837nMmArnyv-wizsIJvc-4_xdgfoUXF4o/view), which proposes the principles as follows:
+
+1. **Declarative Configuration**: All resources managed through a GitOps process must be completely expressed declaratively.
+2. **Version controlled, immutable storage**: Declarative descriptions are stored in a repository that supports immutability, versioning and version history. For example, git.
+3. **Automated delivery**: Delivery of the declarative descriptions, from the repository to runtime environment, is fully automated.
+4. **Software Agents**: Reconcilers maintain system state and apply the resources described in the declarative configuration.
+5. **Closed loop**: Actions are performed on divergence between the version controlled declarative configuration and the actual state of the target system.
```

Comment:
We should not have an outdated version of the principles here IMHO.

Reply:
Agree, we should just remove this now, and simplify the README a lot now that we have the OpenGitOps project that holds all this stuff 👍

Reply:
Resolved by omission